tests/validation/tests/v3_api/test_encryption_secrets.py

```diff
@@ -0,0 +1,48 @@
+from .common import *  # NOQA
+import pytest
+from .test_rke_cluster_provisioning import rke_config
+
+providers = [{"aescbc": {"keys": [{"name": "k-fw5hn", "secret": "RTczRjFDODMwQzAyMDVBREU4NDJBMUZFNDhCNzM5N0I="}]}}, {"identity": {}}]
+custom_resources = [{"resources": ["secrets"], "providers": providers}]
+custom_config = {"apiVersion": "apiserver.config.k8s.io/v1", "kind": "EncryptionConfiguration", "resources": custom_resources}
+SECRET_ENCRYPTION_CONFIG = {"customConfig": custom_config, "enabled": True, "type": "/v3/schemas/secretsEncryptionConfig"}
```

Comment:
Can we have all of this - providers, custom_resources, custom_config, SECRET_ENCRYPTION_CONFIG as one parameter? Can you follow how it is here - https://github.com/rancher/rancher/blob/master/tests/validation/tests/v3_api/test_rke_cluster_provisioning.py#L20 so that it is readable?

Reply:
Addressed in changes.